lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -686,6 +686,9 @@ def commit(request, action, amount, options)
           response = parse(ssl_post(test? ? self.test_url : self.live_url, build_request(request, options)))
         rescue ResponseError => e
           response = parse(e.response.body)
+        rescue REXML::ParseException => e
+          response = {}
+          response[:message] = e.to_s
```

Comment:
@shasum looks good- just FYI when I merge this in I'll probably merge these into a single line like-

```ruby
response = { message: e.to_s }
```